comp/core/status/statusimpl/status.go

```diff
@@ -307,23 +311,25 @@ func (s *statusImplementation) GetStatusBySection(section string, format string,
 			return []byte{}, nil
 		}
 	default:
-		providers, ok := s.sortedProvidersBySection[section]
+		providers, ok := s.sortedProvidersBySection[strings.ToLower(section)]
 		if !ok {
-			return nil, fmt.Errorf("unknown status section '%s'", section)
+			res, _ := json.Marshal(s.sortedSectionNames)
```

Comment:
Do we need to Marshal `sortedSectionNames`? 

Could we do:  
```go
errorMsg := fmt.Sprintf("unknown status section '%s', available sections are: %s", section, strings.Join(s.sortedSectionNames))
```

I have the impression that we could avoid that extra `Marshal` operation

Reply:
That's true that at first sight it seems overkill ...
The problem is that some `section` names contains whitespaces, so it's better to print I think section names between quotes in a JSON array style, this way the user understand clearly where the section name start and end.

To this end, using `json.Marshall` was way more suitable than `strings.Join()` IMO. 

#### Examples
##### Previous suggestion message:
```
Error: unknown status section 'wrong', available sections are: collector, aggregator, apm agent, dogstatsd, endpoints, forwarder, jmx fetch, logs agent, otlp, process agent, remote configuration, transport proxy warnings
```

##### New suggestion message:
```
Error: unknown status section 'wrong', available sections are: ["collector","aggregator","apm agent","dogstatsd","endpoints","forwarder","jmx fetch","logs agent","otlp","process agent","remote configuration","transport proxy warnings"]
```

Reply:
Thanks so much for the reply and the context. 

Agree that having the sections surrounded with quotes makes things easier to read 👌 Let's keep for now then. 

Another thing I just realized that some section names are quite long and error prone for customers to type. Ex. `remote configuration` I think we could think of adding some logic the status provider to provide an alias. In the case of `remote configuration` we could use `rc`. In the end the command would look `status component rc`

Of course this an idea and is not meant to be done in this PR nor in the near future as we still have no data if customers will use this feature 😄